src/imerg/create_auth_files.py

```diff
@@ -0,0 +1,43 @@
+import os
+import platform
+import shutil
+from subprocess import Popen
+import logging
+
+logger = logging.getLogger()
+
+
+def create_auth_files():
+    # script to set credentials from
+    # https://disc.gsfc.nasa.gov/information/howto?title=How%20to%20Generate%20Earthdata%20Prerequisite%20Files
+    IMERG_USERNAME = os.environ["IMERG_USERNAME"]
+    IMERG_PASSWORD = os.environ["IMERG_PASSWORD"]
```

Comment:
Should add to the main `README.md` that these are required. 

Also curious, usually I would use `os.getenv("some-var")` to retrieve an env var. Is there a difference with just `os.environ["some-var"]`? 

You may also need to call `load_dotenv()` to support local development with the `.env` file.

Reply:
Oh, good catch... I can confirm this works the exact same way but for the purposes of keeping the same standard I will change this to use `os.getenv`. It has been working without issues on databricks.

Reply:
Cool! Don't think it's a big deal either way -- was just curious

Reply:
`os.getenv()` returns `None` if the env var isn't defined, whereas `os.environ[]` throws an error (like accessing a `dict` with `.get()` vs. `[]`). So I think my preference is for `os.getenv()` because I think it's nicer to handle the `None` situation instead of the error. I do acknowledge that I originally wrote in the `os.environ[]` line here though 😆